frontend/src/features/seeds/routes/CreateSeed.tsx

```diff
@@ -23,7 +24,14 @@ export function CreateSeed() {
     isLoading: isUploadingSeed,
     isSuccess: isUploadingSuccess,
   } = useMutation(['create new seed'], createSeed, {
-    onError: () => {
+    onError: (error) => {
+      const errorTyped = error as AxiosError;
+
+      if (errorTyped.response?.status === 409) {
+        toast(t('seeds:create_seed_form.error_seed_already_exists'));
```

Comment:
Can you show this in the respective fields instead of the toast?

Reply:
Yes I could, but that might be worse from a usability standpoint than the current solution, since we only use either standard HTML 5 validation or toasts to show errors at the moment.

Reply:
afaik @Bushuo also used zod which is much more than standard HTML 5 validation?

Reply:
So I tried to give feedback through standard HTML validation messages, but encountered a few difficulties.

1. Zod seems to be a client side validation library, but the error that we are displaying here can only be detected on the backend.
2. You can set a custom validation message using a JS-DOM method. 
    However, since we are using react this can only be done using a ref to the input field.
    For some reason getting said ref turned out to be very tricky, and I could not get it to work until now.

Reply:
Thanks for answering the question. Then let us leave the error currently as it is.